CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 13.2.0 (07/05/23)
+
+* Teleport Assist
+  * Added command execution progress feedback and settings UI. [#28480](https://github.com/gravitational/teleport/pull/28480)
+* Server Access
+  * Added option to preserve automatically created host users instead of deleting them. [#28432](https://github.com/gravitational/teleport/pull/28432)
+  * Fixed issue with `tsh join` when per-session MFA is enabled. [#28456](https://github.com/gravitational/teleport/pull/28456)
+* Database Access
+  * Updated `tsh db connect` to prefer `mongosh` client. [#28668](https://github.com/gravitational/teleport/pull/28668)
+  * Fixed issue with database agents not respecting graceful shutdown. [#28369](https://github.com/gravitational/teleport/pull/28369)
+* Device Trust
+  * Improved Jamf config validation. [#28553](https://github.com/gravitational/teleport/pull/28553)
```

Comment:
This is the first release that enables the Jamf integration, so can we just say something like:
```suggestion
  * Add Jamf integration for inventory management
```